### PR TITLE
Replace Unity lesson

### PR DESCRIPTION
**Pull request by mdlincoln**
Re: #717 

This PR will hold the retirement commits for the old lesson (which I've already added) as well as the new text for the revised lesson (still to be added by author/editors).

- [x] Retire old lesson
- [x] Add new text to `/lessons`
- [x] Update retired lesson metadata with proper link to new lesson and any additional retirement description that's desired
- [ ] Conduct light sustainability review.
- [ ] Pin a ES editor so s/he can update the list of pending translations

**Add to this PR by committing to the `replace_unity_lesson` branch. DO NOT OPEN ADDITIONAL PRs**

### CODE REVIEW

**Comment by walshbr** [comment]
@mdlincoln - does @jacobwgreene need to be added to the repository so that he has commit access for this PR? If so, I don't think I have permissions to do so.

**Comment by mdlincoln** [comment]
I've added him as a temporary collaborator

**Comment by arojascastro** [comment]
Sorry to jump into this, but does this mean that lesson is going to be retired and a new updated lesson on the same will be published? Asking so I can update our [pending translation list](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/es/lista-de-traducciones.md)

**Comment by mdlincoln** [comment]
Yes @arojascastro

**Comment by walshbr** [comment]
@arojascastro - yep that's correct. Jump away!

**Comment by arojascastro** [comment]
Dankeschön!

**Comment by jacobwgreene** [comment]
I just pushed changes to the replacement lesson branch. The new lesson is titled "updated-intro-to-ar-development." I also added new images to the images folder under the same name. Let me know if I added things incorrectly, etc.

**Comment by jacobwgreene** [comment]
It looks like I may have made some changes under another account (tracearcs@gmail.com). Let me know if I need to make another commit.

File: lessons/updated-intro-to-ar-development.md
Comment (mdlincoln) [inline lessons/updated-intro-to-ar-development.md]: Let's make sure to update this published date, since this is a new lesson.

File: lessons/updated-intro-to-ar-development.md
Comment (mdlincoln) [inline lessons/updated-intro-to-ar-development.md]: We definitely want to include a banner at the start of this lesson pointing to the retired one. 

To add the banner in here, use:

```html
<div class="alert alert-warning">
    <p>lorem ipsum etc...</p>
</div>
```

File: lessons/updated-intro-to-ar-development.md
Comment (mdlincoln) [inline lessons/updated-intro-to-ar-development.md]: TBH I still think it's confusing to have an identical title for this lesson, and makes for a really bad user interaction with the site. Anyone using this in workshops/classes etc needs to know that it has been _substantially_ changed, so I think we should err on the side of making that change obvious, rather than trying to streamline it away.

File: lessons/updated-intro-to-ar-development.md
Comment (mdlincoln) [inline lessons/updated-intro-to-ar-development.md]: Reminder to add the new reviewers before publishing.

**Comment by walshbr** [comment]
Also just a note - @alsalin is going to review the lesson with an eye to sustainability, and she has a colleague who works on VR that she will bring onboard as well. So we're covered for reviewers. I'll make sure they take a look once we're at a stable state for the lesson.

File: lessons/updated-intro-to-ar-development.md
Comment (walshbr) [reply lessons/updated-intro-to-ar-development.md]: Done in [this commit](https://github.com/programminghistorian/jekyll/pull/800/commits/d77c2bcb01ad182e5d11f9454599105bb74c6e13). I took out the p tags because the padding was causing some weird looking spacing, but let me know if you want them back.

File: lessons/updated-intro-to-ar-development.md
Comment (walshbr) [reply lessons/updated-intro-to-ar-development.md]: Done

File: lessons/updated-intro-to-ar-development.md
Comment (walshbr) [reply lessons/updated-intro-to-ar-development.md]: @jacobwgreene do you have thoughts on another title?

Comment (jacobwgreene) [reply lessons/updated-intro-to-ar-development.md]: @walshbr How about "Creating Mobile Augmented Reality Experiences in Unity."

Comment (walshbr) [reply lessons/updated-intro-to-ar-development.md]: Works for me - good for you @mdlincoln?

Comment (walshbr) [reply lessons/updated-intro-to-ar-development.md]: Think I caught every instance in which changing the title would affect something. Will ping to start the sustainability review now.

**Comment by walshbr** [comment]
@alsalin - you and your colleague should be good to go doing a sustainability review for this lesson. Let us know if you  need someone added to the repo, and let me know if you  have any questions!

**Comment by alsalin** [comment]
My review below! Note that Arya and his wife just had a baby, it seems! So he will likely be unable to complete his review, or at least not at this time.

## **General Notes**
This tutorial is specific to getting up and running with Unity as opposed to introducing a Unity-based methodology. Should Unity undergo a redesign, this tutorial would have to be completely redone. This tutorial also relies heavily on integration with other SW platforms, and if those interfaces are updated, this tutorial will need additional updates. If I were applying sustainability indicators, 1 being stable and 5 likely needing future updates, I would rank this a 5.

## **General Recommendations**
**Readability**
There are many areas where the author describes actions the user should take. For readability purposes, it may be best to break these out to make them easier to read. This is a very long lesson with many steps per section, so any way to increase skimmability would help. Using numbered lists and varied, but consistent, stylings will also help the reader who may be following this lesson while trying all these steps out in Unity.

For example,
"Unity should automatically detect the location of the Java Development Kit once it has finished installing. 

If it does not,
1. Restart Unity and go to **Edit > Preferences**. 
2. Navigate to **External Tools** and ensure that the _JDK_ field is pointing to the Java Development Kit. 
3. Click the **Browse** button next to the _JDK_ field and select the _jdk[version number]_ folder in the `C:/Program Files/Java folder`. Leave the _NDK_ field blank."

**Screenshots**
This lesson should undergo a screenshot audit. While lots of work has gone in to illustrating what needs to be done and what outcomes look like via screenshots, they are a sustainability nightmare. Which screenshots actively assist readers in understanding what to do at any given step and which are simply illustrative of an outcome. 

Instead of using screenshots for every example, consider the above numbered lists for readability.

## **Detailed Observations**
- p1: mention explicitly that unity will be covered
- lesson goals: mention version number that this is based on
- introduction in general: name out all sw dependencies for this lesson. there seem to be several. each dependency should also list the version number that this lesson used.
- L44 good job pointing to unity materials
- L50 HP reveal example - caption for image, include current version number
- L52 first mention of vuforia - define/link
- general question: are there any hardware requirements to do this tutorial?
- L65 version of android SDK needed (if it's put up at the top, that's ok)
- L125 Navigating Unity section: Navigating section could benefit from subheadings. It seems as though screenshots are critical for this section but it is unclear whether one is to do something in this section or merely get an overview of where things are. Could it be possible to use this section to prepare for the next section? e.g. introduce hierarchy panel and then say delete the light and object?
- L127 Section references in bold - e.g. feel free to skip down to “Setting up Unity for AR Development.”
- L147 "For your AR application, you will not need the Main Camera or Directional Light game objects. Select them both, right click (or Ctrl click), and select “Delete.”" is unclear - where do you select these items and delete them?
- L155 random "f"
- L161 vuforia screenshot necessary?
- L193 use gimp internal documentation if possible, not youtube video -https://docs.gimp.org/2.8/en/gimp-tutorial-quickie-crop.html and https://docs.gimp.org/2.8/en/gimp-tutorial-quickie-scale.html  or simply point users to documentation so that they can get the latest version of walkthroughs https://www.gimp.org/docs/

**Comment by walshbr** [comment]
Thanks @alsalin! And no worries about Arya - I think since a light sustainability review was requested that your review is probably more than sufficient. 

Based on the editorial board conversations, my sense is that we do want to keep this lesson in the mix for now, even with the concerns about it having to be rewritten in the future. I think your suggestions will help with some of the sustainability concerns and at least make it easier to revise the lesson in the future if the dependencies do change drastically. The screenshots, for example, definitely seem like a place where @jacobwgreene could intervene. He can correct me if I'm wrong, but I'm not sure there are clear ways to reduce the number of dependencies in this lesson. Clearly noting the version numbers for all dependencies used in the lesson is a good idea, but I wonder if it might be possible to direct people more clearly as to how to download those specific versions as well? In other words, direct people a little more to the specific tech stack used and rely less on the "or later" bit of "2017.2 or later."

@jacobwgreene - I think you had mentioned an interest in writing further lessons on augmented reality, right? If so, it might make sense to patch this for now while incorporating as much of the review suggestions and then keep these larger suggestions about sustainability in mind as you think towards future possibilities. I.e. - what might a lesson on AR look like that relies less on the particular technology and more introduces an approach whose dependencies could be more easily patched as necessary? That way this lesson can still live out a full life until a hypothetical future in which Unity changes more drastically, and we can use it as an example as you put together other ideas. But I'm open to thoughts.

**Comment by acrymble** [comment]
I'm not clear why this discussion is happening on a pull request rather than using our /submissions system.

**Comment by walshbr** [comment]
@acrymble - sorry I think the conversation about whether this was a revision to an extant lesson or a substantially new lesson might have gotten the issue's relationship to the pipeline confused for me. I also might have been confused as to whether the light sustainability review was a full-on re-triggering of the review pipeline. Can you confirm how you'd like us to proceed @acrymble? I can copy files over to the ph-submissions repo, open a ticket on that repo, copy other the review conversation thus far, and continue the conversation there if that makes sense to you as the best approach?

**Comment by acrymble** [comment]
@walshbr you're the editor, so it's up to you how to proceed. I haven't looked at the lesson, so I couldn't comment on how revised it is. I don't think we want to get in the habit of doing submissions in different ways though. That gets confusing.

**Comment by jacobwgreene** [comment]
Thanks for the helpful revision recommendations @alsalin and @walshbr 

Correct me if I'm wrong, but the main concerns seem to be 1) improving the lesson's readability by incorporating more detailed steps and numbered lists, 2) removing screenshots where possible, particularly those that depict version-specific instances of Unity/Vuforia, and 3) looking for any ways that the lesson could be written for more generalized AR development in Unity. 

@walshbr yes, this lesson is highly specific to the software versions used in the lesson. The problem with working with AR in general (and trying to teach others how to use it) is that the tech is constantly changing. So, one solution might be for me to go through the lesson and more clearly separate out areas that would likely remain stable (e.g. navigating the unity interface) from those that might need to be updated more regularly (e.g. adding target images). I will also be sure to clarify what versions of the various SDKs I am using in the program and where they can be accessed on Unity and Vuforia's archive sites.

Do you have a specific timeline when you would like these changes made? I should be able to tackle them in the next few weeks or so. I'm currently prepping for a big move out west this summer where I'm starting a new job in the fall, so I probably won't have much time once June hits.

**Comment by walshbr** [comment]
Thanks, @acrymble, for the helpful thoughts and cautions. I had a second just now, so I diffed the old lesson against the new to see how revised the thing was. While there are a lot of changes and some are longer than others, the content of  virtually all of them boils down to this sort of thing by varying degrees: “Click on X menu and select Y function.” -> “Click on X menu and select Z function.” One section has content and language that are new and different from those sort of cosmetic changes to update for the new version of Unity - [this one](https://deploy-preview-800--ph-dev.netlify.com/lessons/creating-mobile-augmented-reality-experiences-in-unity#create-a-simple-user-interface). By my math that means the lesson only has about 10-15% of content that has substantially changed in a meaningful way - it’s largely the same thing with new skin, hence the PR and not the full review process. 

If we were just going to patch what you have here I’d be inclined to say we should close out the conversation on the PR to minimize confusion. But sounds like @jacobwgreene is willing to take on some more substantial revisions of the lesson in light of @alsalin’s sustainability concerns? If that’s the case, let’s take a step back and open a ticket on the ph-submissions repo as though this were actually a new lesson, as it seems like things will change more drastically. I’ll open a review ticket and get @alsalin (if she’s willing) and another reviewer to take a look when you get things up. It seems like we all agree that it’s more a matter of if than when as to when the rug will be pulled from under this lesson. Lets do a bit more work now to guard against those futures. I like your plan for revisions, @jacobwgreene.

Let me know if that sounds like a plan to you, @jacobwgreene, and I’ll get you set up on the ph-submissions repo. And if that’s the plan, @mdlincoln, might it make sense to go ahead and retire the old lesson right away before the replacement is ready? I can clean up the PR so that it would do only that.

@jacobwgreene - if you’ve got time in the next few weeks that sounds good to me. Might as well get to this now while it’s fresh in your mind.

**Comment by jacobwgreene** [comment]
@walshbr That sounds good to me!

**Comment by mdlincoln** [comment]
@walshbr sure, if you just want to retire the lesson now then let's do that. Make sure to edit the retirement reason field to include whatever text you want to have.

**Comment by walshbr** [comment]
Give that a look @mdlincoln? I think that should be everything we need to retire the one lesson. The one travis build is failing and the other is passing - I _think_ the travis-ci/pr one is the one I should be looking at yes? Should be good to go for closing and merging. 

@jacobwgreene and @alsalin - I opened a new ticket on the ph-submissions repo to continue the discussion. Whenever you're ready, @jacobwgreene, post the materials to that repo and we can go from there! I've got a proposals ticket at the moment, but I'll convert it to a review ticket when ready.

**Review by mdlincoln — COMMENTED** [review]
This looks good! Since I actually opened this PR originally, though, you need to hit approve @walshbr

**Review by walshbr — APPROVED** [review]

**Pull request merged**
